### PR TITLE
Organized keys

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,33 +22,335 @@ Install dependencies `npm install`.
...
+- PM1.0
+- PM2.5
+- PM4.0
+- PM10
+
+-- Electricity Sensor Types
+- Energy
+- LEM
+- External Analog Input
+- do_1_lora
```
Comment (mariusgiger) [inline README.md:55]: what is this?

File: README.md [hunk 2]
```diff
@@ -22,33 +22,335 @@ Install dependencies `npm install`.
...
+- do_2_lora
+- do_1_error
+- do_2_error
+- do_1
+- do_2
+- manually
```
Comment (mariusgiger) [inline README.md:61]: what is this?

File: README.md [hunk 3]
```diff
@@ -22,33 +22,335 @@ Install dependencies `npm install`.
...
+- Gyro
+- Magnetometer
+
+-- GPS Sensor Types
+- Latitude
+- Longitude
+- Altitude
+
+-- People Counting Sensor Types
+- Forward
```
Comment (mariusgiger) [inline README.md:81]: use a more descriptive name, like Forward Count

File: README.md [hunk 4]
```diff
@@ -22,33 +22,335 @@ Install dependencies `npm install`.
...
+- Backwards
+- Gender
+- FaceMask
+- Dwelltime
+- Average Waiting Time
+- Queue Depth
+
+-- Diverse Sensor Types
```
Comment (mariusgiger) [inline README.md:89]: typo Diverse -> Other?

File: README.md [hunk 5]
```diff
@@ -22,33 +22,335 @@ Install dependencies `npm install`.
...
+- Button
+- Distance
+- Vibration
+- Water leak
+- Anomaly
+- Status
+- Image
```
Comment (mariusgiger) [inline README.md:96]: what is this?

File: README.md [hunk 6]
```diff
@@ -22,33 +22,335 @@ Install dependencies `npm install`.
...
+- eastWindSpeed
 - temperature
-- Humidity
-- CO2
-- VOC
-- Barometric Pressure
-- Noise
+- averageTemperature
+- temperaturePT100
+- tempHistory0
```
Comment (mariusgiger) [inline README.md:117]: does not make sense to include that here

File: README.md [hunk 7]
```diff
@@ -22,33 +22,335 @@ Install dependencies `npm install`.
...
+- tempHistory6
+- tempHistory7
+- humHistory0
+- humHistory1
+- humHistory2
+- humHistory3
+- humHistory4
+- humHistory5
+- humHistory6
+- soilMoistureAtDepth0
```
Comment (mariusgiger) [inline README.md:132]: does not make sense to include that here

File: README.md [hunk 8]
```diff
@@ -22,33 +22,335 @@ Install dependencies `npm install`.
...
+- soilMoistureAtDepth1
+- soilMoistureAtDepth2
+- soilMoistureAtDepth3
+- soilMoistureAtDepth4
+- soilMoistureAtDepth5
+- soilMoistureAtDepth6
+- soilMoistureAtDepth7
+- soilTemperatureAtDepth0
```
Comment (mariusgiger) [inline README.md:140]: does not make sense to include that here

File: README.md [hunk 9]
```diff
@@ -22,33 +22,335 @@ Install dependencies `npm install`.
...
+- Vibration
+- Water leak
+- Anomaly
+- Status
+- Image
+
+
+## Available data keys
+
+The following available data keys should be used.
```
Comment (mariusgiger) [inline README.md:101]: does it really make sense to include all data keys here, I would opt for just declaring the most common ones

**Review by mariusgiger — CHANGES_REQUESTED** [review]
please cleanup the README it does not make sense to clutter it with data keys

File: README.md (hunk 9 above)
Comment (J2thatsme) [reply README.md:92]: Cutted some keys. If there are more changes required maybe it would be good to have a call.

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**